docs/data/data-grid/row-spanning/RowSpanningCustom.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+
+export default function RowSpanningCustom() {
+  return (
+    <Box sx={{ height: 350, width: '100%' }}>
+      <DataGrid
+        rows={rows}
+        columns={columns}
+        showCellVerticalBorder
+        showColumnVerticalBorder
+        disableRowSelectionOnClick
+        hideFooter
+        unstable_rowSpanning
+        sx={{
+          '& .MuiDataGrid-row:hover': {
+            backgroundColor: 'transparent',
+          },
+        }}
+      />
+    </Box>
+  );
+}
+
+const columns: GridColDef[] = [
+  {
+    field: 'name',
+    headerName: 'Name',
+    width: 200,
+    editable: true,
+  },
+  {
+    field: 'designation',
+    headerName: 'Designation',
+    width: 200,
+    editable: true,
+  },
+  {
+    field: 'department',
+    headerName: 'Department',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'age',
+    headerName: 'Age',
+    type: 'number',
+    width: 100,
+    valueGetter: (value) => {
```

Comment:
```suggestion
    valueFormatter: (value) => {
```

To fix sorting for this column